conans/client/tools/oss.py

```diff
@@ -351,7 +355,7 @@ def get_gnu_triplet(os_, arch, compiler=None):
             machine = "powerpc64le"
         elif "ppc64" in arch:
             machine = "powerpc64"
-        elif "powerpc" in arch:
```

Comment:
was this a bug? or `powerpc` is never encountered as `uname` value?

Reply:
This is not related with `uname`, right? 😕 It is searching for `powerpc` in the conan archs, but there is no `powerpc` arch

Reply:
yeah, it wasn't obvious due to the usage of `aarch64` here. thanks for the clarification.